internal/cephfs/volume.go

```diff
@@ -34,6 +34,15 @@ var (
 	// Subvolume group create gets called every time the plugin loads, though it doesn't result in error
 	// its unnecessary
 	cephfsInit = false
+
+	// ceph fs subvolume resize is supported from 14.2.8 onwards
+	maxCephVersionForResize      = 14
+	patchCephVersionForResize    = 2
```

Comment:
@Madhu-1 major,minor, patch would be good I believe insteda of max, patch, and subLevel